### ISSUE TITLE
Can not delete branch that I created 

### ISSUE THREAD

**Issue by alyssa51402** [FlutterFlow/flutterflow-issues#3693]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

I have created a branch within my project. After using the branch, I realize there was nowhere to delete it. The person who runs the project is able to delete all the branches. I should be able to delete my own and cannot. I couldn't find anything in the flutter docs that gave reasoning for this either. 

### Expected Behavior

I expected to see a delete branch button or option. 

### Steps to Reproduce

1. Create a branch 
2. delete branch (no delete option) 

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT40hs/17ad/s7h97ZfyWvpUoD47HEwMUOYzpu1RaCosfZT2BLMcec/RVERvQsDmSmxmPGKJ/GoW0uj7i9nxEPVcISuaf4hb+LVQcj3LZE6tMYy2FJWvdHFQHcNbGlyf4J6RgBB/Kslocl4b3U2lf9+sd3XZJqG2NjA4Dq/LZO4=

### Visual documentation

<img width="1439" alt="Screenshot 2024-08-26 at 2 10 43 PM" src="https://github.com/user-attachments/assets/8e6a0114-5e57-4fb4-92bd-ad5c7f9e9f13">

### Environment

```markdown
- FlutterFlow version: V4.1.83
- Platform: Web
- Browser name and version: Chrome
- Operating system and version affected: macOS 14.5
```

**Comment by rzambroni** [comment]
Hi @alyssa51402,

I was unable to replicate the issue on any of my projects. However, I have seen this happen once before, and it resolved itself after restarting the app. Are you still unable to see the 'Close Branch' option? Does this issue occur for all branches or just for that particular one?

Thanks for your help.

**Comment by alyssa51402** [comment]
Hi, thank you for responding. I don't the 'Close Branch' option for any of the branches. I have tried a few times on multiple days and have not seen the option to 'Close Branch. I am just trying to close one that I created. - Alyssa Sigler 
    On Tuesday, August 27, 2024 at 09:06:43 AM EDT, rzambroni ***@***.***> wrote:  
 
 


Hi @alyssa51402,

I was unable to replicate the issue on any of my projects. However, I have seen this happen once before, and it resolved itself after restarting the app. Are you still unable to see the 'Close Branch' option? Does this issue occur for all branches or just for that particular one?

Thanks for your help.

**Comment by rzambroni** [comment]
Hey Alyssa, would you mind adding [josef@flutterflow.io](mailto:josef@flutterflow.io) as a collaborator to your project? we want to take a deeper look to your project.<br>Thanks in advance.

**Comment by alyssa51402** [comment]
Hi, I don't believe I have access to add a collaborator since I am not the owner of the project. I will reach out to them about adding him as a collaborator. Thank you, Alyssa Sigler
    On Wednesday, August 28, 2024 at 02:07:03 PM EDT, rzambroni ***@***.***> wrote:  
 
 


Hey Alyssa, would you mind adding ***@***.*** as a collaborator to your project? we want to take a deeper look to your project.
Thanks in advance.

**Comment by alyssa51402** [comment]
I had the email you sent me, added as a collaborator for the project. 
    On Thursday, August 29, 2024 at 05:02:24 PM EDT, Alyssa S ***@***.***> wrote:  
 
  Hi, I don't believe I have access to add a collaborator since I am not the owner of the project. I will reach out to them about adding him as a collaborator. Thank you, Alyssa Sigler
    On Wednesday, August 28, 2024 at 02:07:03 PM EDT, rzambroni ***@***.***> wrote:  
 
 


Hey Alyssa, would you mind adding ***@***.*** as a collaborator to your project? we want to take a deeper look to your project.
Thanks in advance.

**Comment by alyssa51402** [comment]
Hello, I heard no update about my issue report for #3693. I added the collaborator ***@***.***  to our flutterflow project. Once I did that, I never heard anything back and 6 days later I received an email saying my issue was closed for inactivity. My issue is still not solved and I am unable to close/delete I branch that I myself created. Thank you, Alyssa Sigler 
    On Wednesday, August 28, 2024 at 02:07:03 PM EDT, rzambroni ***@***.***> wrote:  
 
 


Hey Alyssa, would you mind adding ***@***.*** as a collaborator to your project? we want to take a deeper look to your project.
Thanks in advance.

**Comment by rzambroni** [comment]
Hello Alyssa,

I apologize for not updating you sooner regarding this report.

We reviewed the permissions with the collaborator you added, who is supposed to have the same editor permissions as you. Interestingly, they were able to close his own branches without any issues, which suggests that something unusual might be happening with your specific permissions on this project.

![image.png](https://uploads.linear.app/3241089d-736a-4422-900b-051e72accdbe/b9b1e014-e982-4a75-8c2b-577380acb975/df4cb453-00ce-439d-9372-a667a0a3d973?signature=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwYXRoIjoiLzMyNDEwODlkLTczNmEtNDQyMi05MDBiLTA1MWU3MmFjY2RiZS9iOWIxZTAxNC1lOTgyLTRhNzUtOGMyYi01NzczODBhY2I5NzUvZGY0Y2I0NTMtMDBjZS00MzlkLTkzNzItYTY2N2EwYTNkOTczIiwiaWF0IjoxNzI2NDk4ODQ0LCJleHAiOjMzMjk3MDU4ODQ0fQ.aVT8Wm6NaLTbbLz7l6t7vfNQAV1sxWeNzLGMDVdcTuE "=294x")

Unfortunately, we don’t have the tools to further investigate the issue on our end. The best course of action would be to reach out to FlutterFlow support directly, as they can take a deeper look into the project and user configuration to help resolve this. You can contact support either via email at **support@flutterflow.io** or by using the in-app chat feature inside of FlutterFlow.

Thank you for your understanding, and I hope this gets resolved quickly!

**Comment by alyssa51402** [comment]
Thank you for the update, I will take a look at the project again and I will reach out to flutterflow support, if needed. - Alyssa Sigler
    On Thursday, September 19, 2024 at 02:11:00 PM EDT, rzambroni ***@***.***> wrote:  
 
 


Closed #3693 as not planned.
